### ISSUE TITLE
Create Frequency Table (2)

### ISSUE THREAD

**Issue by aliyahtbureau** [aliyahtbureau/QMB210-17-2#3]

**Comment by aliyahtbureau** [comment]
On Campus or Off Campus vs. What meal plan if any and why

**Comment by sophieheinan** [comment]
Should we count blank answers as a part of the frequency? In other words, would no answer be included in the total # of responses?

**Comment by aliyahtbureau** [comment]
@sophieheinan I believe only the data that is fully complete - I can double check with Jerry as well and let you know for sure.

**Comment by aliyahtbureau** [comment]
@sophieheinan just talked to jerry - only real complete data! we can delete all the not complete ones

**Comment by sophieheinan** [comment]
Okay, I did not include the incomplete data. I have made the frequency tables if you want to work on the importance explanation @graceshaffer that would be great. I have attached my Excel worksheet and have a box already made for you to do an explanation (Tab labeled "Frequency Table 2"). Let me know if you can't access it.

[UT Dining Survey_October 11, 2023_19.48.xlsx](https://github.com/aliyahtbureau/QMB210-17-2/files/12894073/UT.Dining.Survey_October.11.2023_19.48.xlsx)

**Comment by graceshaffer** [comment]
Sorry, just seeing this. I can access it, do you if there is any specific requirements for the explanation?

**Comment by graceshaffer** [comment]
I will look at the rubric to be sure but was just wondering if he gave you guys any specifics on it.

**Comment by sophieheinan** [comment]
I've been going off the rubric. On slide 11 there is an example of an explanation otherwise this is what the interpretation should cover: "Why did you make these tables? What do they tell us about your topic?
Why is it important? What conclusions can we draw? What additional questions might this raise?" I also have one completed on a different tab (on Excel) that you can use as an example. Hope this helps, let me know if you have any other questions.

**Comment by graceshaffer** [comment]
Okay, thank you! Sounds great. I will have that done within the hour.

**Comment by graceshaffer** [comment]
Please let me know if I should change anything. I didn't include any specific data related to what meal plan students had. I primarily talked about living on or off campus versus having or not having a meal plan. I wasn't sure how in depth he wants it. So just let me know if you think I should change that or take anything out.

Thanks

**Comment by sophieheinan** [comment]
I'm not seeing anything on my end so I'm going to send you the excel worksheet through email so you can copy and paste your work. That way it should be collaborative when you open it and not just making a copy. Sorry for the confusion

**Comment by graceshaffer** [comment]
Okay, no problem!

**Comment by graceshaffer** [comment]
Are you sending it to my UT email?

**Comment by sophieheinan** [comment]
Yes, I sent it to your UT email. Let me know if you received it and were able to edit it. If not you can just copy your response in here and I can add. Sorry about the confusion, excel isn't the easiest program to work with others on.

**Comment by graceshaffer** [comment]
I don't see it, my email is grace.shaffer2@spartans.ut.edu

**Comment by graceshaffer** [comment]
Got it, thanks.

**Comment by sophieheinan** [comment]
Your explanation looks good to me, nice work!

**Comment by graceshaffer** [comment]
thank you

**Comment by graceshaffer** [comment]
Hey I just want to make sure that my part is done, and wanted to check if theres anything else you need me to do? @aliyahtbureau

**Comment by aliyahtbureau** [comment]
@graceshaffer I think you're good! I see you've been working with @sophieheinan, as long as she approved and looked at your work and said it was okay, then i'm okay! Do you have any questions/ do you feel good for the final video and explanations?

**Comment by sophieheinan** [comment]
I feel very confident with the frequency tables and the interpretation

**Comment by aliyahtbureau** [comment]
@sophieheinan great news can't wait to see them!!

**Comment by graceshaffer** [comment]
For the final video I am just elaborating on the importance/explanation that I already wrote for frequency table 2?

**Comment by aliyahtbureau** [comment]
@graceshaffer exactly - there are specific questions if you are worried in the rubric under assignments

**Comment by graceshaffer** [comment]
Perfect, thank you !!
